buy-resell/products/3rd-party/FinanzOnline Management.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: /posdealers/buy-resell/products/3rd-party/FinanzOnline Management
+title: FinanzOnline Management
+---
+# FinanzOnline Management
+
+:::info summary
+
+After reading this, you will understand the benefits for PosOperators of connecting their fiskaltrust.Account to _FinanzOnline_ and how PosDealers can set up the integration.
+
+:::
+
+:::caution Germany / France
+
+As _FinanzOnline Management_ is only available in Austria, this tutorial does not apply to Germany or France.
+
+:::
+
+## FinanzOnline: Overview
+
+The legal basis for FinanzOnline in Austria was created with the FinanzOnline ordinances in 2006.
+FinanzOnline provides electronic access to the tax authorities for entrepreneurs and their representatives. FinanzOnline offers round-the-clock access to relevant tax documents without the need for additional software and provides the services of the company service portal (USP), such as _e-Rechnung_.
+FinanzOnline is constantly being developed into an information and transaction portal in tax matters.
+
+### FinanzOnline: Prerequisites for PosOperators
+
+You as a PosDealer or as a PosOperator start with creating or collecting the access data of a FinanzOnline user who is authorized for the cash register web service.
+After your login to fiskaltrust.Portal, clarify the existing entitlements for the chosen fiskaltrust account (see `Shop` / `Entitlements` for _AT_FON_NOTIFICATION_).
+Switch to `Tools` / `FinanzOnline notifications` and register your FinanzOnline data, for further instructions, see [our austrian portal-manual](https://docs.fiskaltrust.cloud/de/docs/portal-manual/austria/fiscalization#enter-and-check-the-finanzonline-access-data-of-the-posoperator) or from [BMF: Anmeldung FinanzOnline](https://finanzonline.bmf.gv.at/eLearning/BMF_Handbuch_Registrierkassen.pdf).  
+
+## FinanzOnline management: 
+
+### Prerequisites for PosDealers 
+
+The fiskaltrust.Portals shows all connected PosOperators at `PosDealer` / `FinanzOnline Management` - the connection between PosDealer and PosOperator is activated via signing the contract. Further at least a SCU must be created. If you as PosDealer miss a PosOperator on this list, select `PosOperator` / `Invitations` to check the invitation status or `PosOperator` / `Overview` for further investigations. 
+
+
+:::warning
+
+Do not use the button `Execute FinanzOnline status update for all PosOperators` on at `PosDealer` / `FinanzOnline Management` - the page already shows the current status, for further information see below.
```

Comment:
if the posoperator makes any changes manually in FOn, we won't see the correct current status

Reply:
will explain this at troubleshooting and Send notifications for all PosOperators - seems to be important enough to be mentioned twice - thank you!